src/Components/Footer/Footer.jsx

```diff
@@ -3,7 +3,7 @@ import './Footer.css';
 import { Link } from 'react-router-dom';
 import { Popup } from 'semantic-ui-react';
 import FooterModal from './FooterModal';
-import { Link } from 'react-router-dom';
+
 
```

Comment:
Useless empty lines + I reiterate : what's happening with your Prettier ? Fix this, on P3 I'll refuse any unlintd PR without reading it